packages/mds-agency/api.ts

```diff
@@ -17,7 +17,6 @@
 import express from 'express'
 
 import logger from '@mds-core/mds-logger'
-import { isProviderId } from '@mds-core/mds-providers'
```

Comment:
I think we should get rid of `isProviderId` altogether. Eventually, we should aim to get rid of the entire `mds-providers` package.

Reply:
I was just gonna leave it around for posterity, but happy to nuke it.

Reply:
Nuked in 532cca4